plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/OpenshiftUserTokenProvider.java

```diff
@@ -43,8 +48,23 @@
 
   @Inject
   public OpenshiftUserTokenProvider(
```

Comment:
Will it be relevant to use `che.fabric8.standalone`. Currently it is a bit unclear in which case which config is used

Reply:
I fixed this in commit https://github.com/redhat-developer/rh-che/pull/711/commits/80846aa6d7d128530093ff874446af01162af0d9